src/config-schemas/openmrs-esm-patient-registration-schema.ts

```diff
@@ -0,0 +1,34 @@
+import { validators } from '@openmrs/esm-module-config';
+export const esmPatientRegistrationSchema = {
+  personAttributes: {
+    arrayElements: {
+      subTitle: { validators: [validators.isString] },
+      label: { validators: [validators.isString] },
+      uuid: { validators: [validators.isString] },
+      name: { validators: [validators.isString] },
+      placeholder: { validators: [validators.isString] },
+      id: { validators: [validators.isString] },
+      validation: { validators: [validators.isString] },
```

Comment:
`validation` looks like it has expected structure, so let's validate that structure. Also, the validator provided doesn't make sense for that structure. It should be something like:

```javascript
validation: {
  required: { default: false, validators: [validators.isBoolean] }
}

Reply:
Done